src/com/theironyard/novauc/People.java

```diff
@@ -0,0 +1,15 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by macbookpro on 2/7/17.
+ */
+public class People {
+String name;
+    boolean famous;
+int age;
+String nationality;
+String profession;
+
```

Comment:
Good job Al.  Variable naming convention was followed well. Class names should be singular, so Person not People.  Let's line up the variables horizontally next time it will be easier to read.